man/systemd.network.xml

```diff
@@ -193,6 +193,16 @@
           This happens when multicast routing is enabled.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>IPv6LinkLocalAddressGenerationMode=</varname></term>
+        <listitem>
+          <para>Specifies how IPv6 link local address is generated. Takes one of <literal>eui64</literal>,
+          <literal>none</literal>, <literal>stable-privacy</literal> and <literal>random</literal>.
+          When unset, the kernel's default will be used. Note that if <varname>LinkLocalAdressing=</varname>
+          is configured as <literal>ipv6</literal> then <varname>IPv6LinkLocalAddressGenerationMode=</varname>
+          is ignored.</para>
```

Comment:
I still don't understand this. Isn't it only used when LinkLocalAddressing= is on, i.e. the opposite of what this says?

Reply:
Ouch my mistake. Don't know what I was thinking.  It should be configured when  LinkLocalAdressing is set and extends .